imports/api/courses.js

```diff
@@ -600,6 +601,15 @@ Meteor.methods({
     return Meteor.call('sessions.delete', courseId, sessionId)
   },
 
+  'courses.getCourseApproved' (courseid) {
```

Comment:
Change the name of the method to be more explicit, e.g. courseRequiresApprovedQuestions() returns true of false